website/content/api-docs/secret/transit.mdx

```diff
@@ -91,6 +91,155 @@ $ curl \
     http://127.0.0.1:8200/v1/transit/keys/my-key
 ```
 
+## Import Key
+
+This endpoint imports existing key material into a new transit-managed encryption key.
+To import key material into an existing key, see the `import_version/` endpoint.
+
+### Parameters
```

Comment:
I believe we are missing the [context](https://github.com/hashicorp/vault/blob/main/builtin/logical/transit/path_import.go#L77-L83) parameter.